crates/cxx-qt-gen/src/parser/qobject.rs

```diff
@@ -124,63 +121,22 @@ impl ParsedQObject {
             &qobject_ty.attrs[attr_index],
             AttributeDefault::Some(|span| LitStr::new("", span)),
         )?;
-        let qml_uri = attrs_map.get(&quote::format_ident!("qml_uri"));
-        let qml_version = attrs_map.get(&quote::format_ident!("qml_version"));
-        let qml_name = attrs_map.get(&quote::format_ident!("qml_name"));
```

Comment:
i would note that we have lost `qml_name` support in this, i think this was for `QML_NAMED_ELEMENT(T)`. But this can be readded in another pull request i guess ?
```cpp
class SqlEventDatabase : public QObject
{
    Q_OBJECT
    QML_NAMED_ELEMENT(EventDatabase)

    // ...
};
```

https://doc.qt.io/qt-6/qqmlengine.html#QML_NAMED_ELEMENT

Reply:
Ah i see further down this is combined into `qml_element = "T"`